### PR TITLE
develop to master

### PR DESCRIPTION
**Pull request by abhishek-1-simform**
develop to master

### CODE REVIEW

File: ExpandableListView/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            isMinifyEnabled = false
+            proguardFiles(
+                getDefaultProguardFile("proguard-android-optimize.txt"),
+                "proguard-rules.pro"
+            )
+        }
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
```
Comment (yashwantgowla-simform) [inline ExpandableListView/build.gradle.kts:44]: Need to update this to Java 11 for jitpack

Comment (yashwantgowla-simform) [reply ExpandableListView/build.gradle.kts:44]: Create a separate PR to develop branch for this fix

File: app/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        release {
+            isMinifyEnabled = false
+            proguardFiles(
+                getDefaultProguardFile("proguard-android-optimize.txt"),
+                "proguard-rules.pro"
+            )
+        }
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
```
Comment (yashwantgowla-simform) [inline app/build.gradle.kts:34]: Change here as well

File: ExpandableListView/build.gradle.kts (hunk 1 above)
Comment (abhishek-1-simform) [reply ExpandableListView/build.gradle.kts:44]: Updated.

**Review by yashwantgowla-simform — APPROVED** [review]

**Pull request merged**